### DISCUSSION TITLE
Synching Audio from a mic and video from OpenCV

### DISCUSSION THREAD

**Discussion by BoltzmannEntropy** [bluenviron/mediamtx#879] [Questions]
Hello,
In my setup, I have audio from a microphone in one stream which I am processing with portaudio, and OpenCV frames from another stream. I would like to synch both streams into a single ffmpeg video. How can this be done? 

Thanks,

**Comment by aler9** [comment]
You can publish the two tracks separately to the server, merge them with ffmpeg and publish the result to the server:

```
ffmpeg -i rtsp://localhost:8554/audio-track -i rtsp://localhost:8554/video-track -map 0:a:0 -map 1:v:0 -c copy -f rtsp rtsp://localhost:8554/merged-stream
 ```

[a comment by an automated account is not shown]
